tests/model_hub_tests/pytorch/requirements.txt

```diff
@@ -26,3 +26,39 @@ transformers
 wheel
 PyYAML
 kornia
+
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly/
+--extra-index-url https://download.pytorch.org/whl/cpu
+# we need at least openvino 2024.2
+--pre
+openvino
```

Comment:
I believe we don't need it in the requirements file

Reply:
Ok, removed.